src/fastnetmon.cpp

```diff
@@ -3581,20 +3581,29 @@ void produce_dpi_dump_for_pcap_dump(std::string pcap_file_path, std::stringstrea
     // Buffer for packets
     char packet_buffer[pcap_header.snaplen];
 
-    unsigned int total_packets_number = 0;
-
+    uint64_t total_packets_number = 0;
     uint64_t dns_amplification_packets = 0;
     uint64_t ntp_amplification_packets = 0;
     uint64_t ssdp_amplification_packets = 0;
     uint64_t snmp_amplification_packets = 0;
 
+    struct ndpi_id_struct *src = NULL;
```

Comment:
I'm not sure about correctness of this code because we need to keep state for whole file. Per packet initilization is useless because we lost state about tcp / flows.